### PR TITLE
Make things green

### PR DESCRIPTION
**Pull request by TheRishabhB**
device-2018-07-15-003451

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (nishatoma) [inline .idea/misc.xml:28]: Why did we downgrade JDK?

Comment (TheRishabhB) [reply .idea/misc.xml:28]: Autogen, I didn't touch that. Studio did it for me.

Comment (nishatoma) [reply .idea/misc.xml:28]: It did it for me locally, I just didn't push it to remote.

Comment (TheRishabhB) [reply .idea/misc.xml:28]: If studio is doing it, it's doing it for a reason.

**Pull request merged**